src/components/ProductDetailCard/ProductDetails.tsx

```diff
@@ -0,0 +1,73 @@
+import { Product } from '../../utils/dataTypes/product';
+import ReviewsList from '../ReviewsList/ReviewsList';
+import PlusMinusButton from '../PlusMinusButton/PlusMinusButton';
+import { useCart } from '../../ShoppingCartContext';
+import './ProductDetailCard.css';
+
+type ProductDetailCardProps = {
+	product_id: Product['product_id'];
+	product_name: Product['product_name'];
+	product_author: Product['product_author'];
+	category: Product['category'];
+	product_description: Product['product_description'];
+	price: Product['price'];
+	product_image: Product['product_image'];
+};
+
+export function ProductDetailCard({
+	product_id,
+	product_name,
+	product_author,
+	category,
+	product_description,
+	price,
+	product_image,
+}: ProductDetailCardProps) {
+	const { handleRemoveFromCart, handleAddToCart } = useCart();
+
+	return (
+		<>
+			<h2> See the book in more detail!</h2>
+			{product_name ? (
+				<>
+					<>
+						<div className='buttons'>
+							<div className='alignBtnWText'>
+								<span>Add 1 to cart</span>
+								<PlusMinusButton
+									buttonText='+'
+									buttonClick={() => handleAddToCart(product_id)}
+								/>
+							</div>
+							<div className='product-image'>
+								<img style={{ maxHeight: 200 }} src={'/' + product_image} />
+							</div>
+							<div className='alignBtnWText'>
+								<span>remove 1 from cart</span>
+								<PlusMinusButton
+									className='buttons'
+									buttonText='-'
+									buttonClick={() => handleRemoveFromCart(product_id)}
+								/>
+							</div>
+						</div>
```

Comment:
You've done a great job getting this to work with the cart! If there's time to refactor, I think it might be nice to put the buttons below the price and lay out as below, with a state monitoring the number in the centre:

![image](https://github.com/fac29/hoda-e-commerce-frontend/assets/127022084/7e4b2c12-315b-4aef-b617-4f62978cca4b)